charts/prometheus-node-exporter/Chart.yaml

```
@@ -6,7 +6,7 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 3.2.0
+version: 3.2.1
```

new feature should raise minor version

```suggestion
version: 3.3.0
```

---

done

---

done